lib/glimesh/streams/channel.ex

```diff
@@ -53,14 +53,14 @@ defmodule Glimesh.Streams.Channel do
   def start_changeset(channel, attrs \\ %{}) do
     channel
     |> changeset(attrs)
-    |> put_change(:status, "live")
+    |> force_change(:status, "live")
```

Comment:
The theory with the force_change is that we might be mutating a bad changeset. This would force the SQL to update even if the changeset is wrong.